.testdata/sample-versions.txt

```diff
@@ -0,0 +1,17 @@
+# WARNING: file generated by gnomes DO NOT EDIT
+#
+# These are versions which are passed to runtests, on each architecture, for CI
+# testing. These do not need to be authoritative.
+#
+# Listing all versions ever provides for quadratic test runtime as we expand
+# architectures and OSes. So we no longer do that.
+#
+# Instead we include all versions of the current release series plus two.
```

Comment:
This appears to be wrong, since we're including a total of five minors, and _not_ all versions, but only the latest version of each of those series.

And the same then applies to the internal stub copy of this text.

Reply:
Thank you for this! I was interpreting "current release series" to mean the three _supported_ minor releases, but that's a lot of implied meaning. I'll reword to be more clear 👍🏼